client/src/components/App/Home/MakeBooking.tsx

```diff
@@ -608,7 +617,7 @@ const MakeBooking: React.FC<Props> = (props) => {
               </OurButton>
               <OurButton
                 onClick={() => {
-                  if (config?.reasonToBookRequired) {
+                  if (config?.reasonToBookRequired && !isAutoApprovedUser) {
```

Comment:
Rather than comparing both variables multiple times, you could maybe combine the logic into a single variable?

e.g. `isReasonRequired` which is the result of `config?.reasonToBookRequired && !isAutoApprovedUser`.  Then you just use the latter for all of these checks?

Reply:
Changes made.